packages/components/psammead-bulletin/src/index.stories.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import notes from '../README.md';
+import { RadioBulletin, TVBulletin, BulletinImage } from './index';
+
+storiesOf('Components|Bulletin', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
+  .add(
+    'Audio',
+    ({ text, script, service, dir }) => {
+      const liveIndicator = boolean('Live', false) ? 'Live ' : null;
```

Comment:
could this be replaced with `&&`? 
```suggestion
      const liveIndicator = boolean('Live', false) && 'Live ';
```